configure.ac

```diff
@@ -23,38 +23,59 @@ LT_PREREQ([2.2])
 LT_INIT([dlopen])
 
 AC_ARG_ENABLE([dependency-checks],
-              AS_HELP_STRING([--disable-dependency-checks], [Disable dependency checks]))
+              AS_HELP_STRING([--disable-dependency-checks],
+                             [Disable dependency checks]))
 AM_CONDITIONAL([DEPENDENCY_CHECKS], [test "x$enable_dependency_checks" != "xno"])
 
 AC_ARG_ENABLE([developer],
-            AS_HELP_STRING([--enable-developer], [Check for and enable tooling required only for SILE developers]))
+            AS_HELP_STRING([--enable-developer],
+                           [Check for and enable tooling required only for SILE developers]))
 AM_CONDITIONAL([DEVELOPER], [test "x$enable_developer" = "xyes"])
 
+AC_ARG_ENABLE([font-variations],
+            AS_HELP_STRING([--disable-font-variations],
+                           [Disable support for OpenType variations and variable fonts that requires very recent HarfBuzz]))
```

Comment:
Wording like `very recent` does not age well, we are probably better off spitting the required HarfBuzz version (and to avoid repeating that, we can move the required versions to variables at the top of the file, many projects do this and it helps grouping them in one place).

Reply:
Good call.